features/steps/when.py

```
@@ -15,6 +15,10 @@ def run(context: Context, command: str):
     if hasattr(context, "home_dir"):
         # Override the HOME variable
         env["HOME"] = context.home_dir
+    env["COMSPEC"] = os.environ.get("COMSPEC", "cmd.exe")
+
+    if os.name == "nt" and not "COMSPEC" in os.environ:
```

Ruff (0.8.2)

Test for membership should be `not in`

Convert to `not in`

(E713)